api/src/main/java/jakarta/data/page/PageRequest.java

```diff
@@ -88,7 +88,7 @@ public interface Pageable<T> {
      * as follows:</p>
      *
      * <pre>
-     * {@code Pageable<Car>} page1Request = Order.by(_Car.price.desc(),
+     * {@code PageRequest<Car>} page1Request = Order.by(_Car.price.desc(),
      *                                       _Car.mileage.asc(),
      *                                       _Car.vin.asc())
      *                                   .page(1)
```

Comment:
Make the indentation align after the rename:
```suggestion
     * {@code PageRequest<Car>} page1Request = Order.by(_Car.price.desc(),
     *                                          _Car.mileage.asc(),
     *                                          _Car.vin.asc())
     *                                      .page(1)
```